shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RQLStatement.g4

```diff
@@ -61,4 +61,5 @@ tableName
     
 keyGeneratorName
     : IDENTIFIER | STRING
-    ;
\ No newline at end of file
+    ;
+    
```

Comment:
Sorry, the blank line at the end of the file seems wrong here, please check if it is caused by indentation.

Reply:
@yx9o  Maybe caused by `tab`